databricks/koalas/series.py

```diff
@@ -2918,47 +2918,56 @@ def rank(self, method='average', ascending=True):
 
         Examples
         --------
-        >>> df = ks.DataFrame({'A': [1, 2, 2, 3], 'B': [4, 3, 2, 1]}, columns= ['A', 'B'])
-        >>> df
-           A  B
-        0  1  4
-        1  2  3
-        2  2  2
-        3  3  1
-
-        >>> df.rank().sort_index()
-             A    B
-        0  1.0  4.0
-        1  2.5  3.0
-        2  2.5  2.0
-        3  4.0  1.0
+        >>> s = ks.Series([1, 2, 2, 3], name='A')
+        >>> s
+        0    1
+        1    2
+        2    2
+        3    3
+        Name: A, dtype: int64
+
+        >>> s.rank()
```

Comment:
Don't we need to sort the result anymore?
Because i think koalas doesn't guarantee the index order internally although now all tests are passed.
How about cc @ueshin @HyukjinKwon

Reply:
For more information, `rank` is sorting the order internally like below.

<img width="367" alt="스크린샷 2019-11-21 오전 8 57 30" src="https://user-images.githubusercontent.com/44108233/69288463-008cfe80-0c3d-11ea-9be5-adb0186803d8.png">

`_scol` means spark column, since koalas manage the data on spark internally.

```python
>>> kser = ks.Series([1, 2, 3, 4, 5])
>>> kser._scol
Column<b'0'>
>>> type(kser._scol)
<class 'pyspark.sql.column.Column'>
```

Reply:
Thank you for explanation.

And I think we need sorting to rank at DataFrame.
But as you know in Series, the sorting operation is already applied in rank method.